### PR TITLE
Skip host stacks for master disable termination

### PR DESCRIPTION
**Pull request by pipo02mix**
[I have found my fix did not work for host account](https://circleci.com/gh/giantswarm/ci-cleaner/27833) because indeed there is no master instance. Skipping that case

### CODE REVIEW

File: pkg/cleaner/aws/aws.go [hunk 1]
```diff
@@ -253,6 +255,17 @@ func stackShouldBeDeleted(stack *cloudformation.Stack) bool {
 	return false
 }
 
+func isGuestStack(stack *cloudformation.Stack) bool {
+	outputs := stack.Outputs
+	for _, o := range outputs {
+		if *o.OutputKey == "MasterImageID" {
+			return true
```
Comment (pipo02mix) [inline pkg/cleaner/aws/aws.go]: It is the best way I found to identify it is a guest stack

**Review by xh3b4sd — APPROVED** [review]
That is probably good enough. Thanks for holding the line.

File: pkg/cleaner/aws/aws.go (hunk 1 above)
Comment (rossf7) [reply pkg/cleaner/aws/aws.go]: If it's easy to change can we make this `isTenantStack`? If not don't worry.

**Review by rossf7 — APPROVED** [review]
Good catch just a suggestion on naming.

**Pull request merged**